### DISCUSSION TITLE
how to save video files from response input

### DISCUSSION THREAD

**Discussion by akshaykadam771** [fastapi/fastapi#6930] [Questions] [answered]
i am working on object detection task. for that i was passing video input from html page to app but i am not able to save that video file in application. How can i save video input file which is coming from html response?

**Comment by ycd** [comment, marked as answer]
Please follow the issue template and edit the question.

**Comment by tiangolo** [comment]
Thanks for the help @ycd ! 🍰 

Sorry @akshaykadam771, I (and we) can't help you if you don't follow the issue template and provide more information.

If you still have problems please create a new issue and follow the template step by step.

Given that, I'm gonna close this one. ☕ 

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
